src/transform-data.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
```

Comment:
If we want to do another major bump, we should get this of this class entirely

Reply:
Sorry Yasserf, you mean eliminate `'use strict'` of all modules in the connector? If that's so, why?

Reply:
No I meant remove the transform-data file entirely! The V4 storage API was made in order to remove the terrible ds id nesting that was introduced at the beginning.